fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -620,14 +668,31 @@ public static FHIRSearchContext parseQueryParameters(Class<?> resourceType,
                         mod           = parameterName.substring(parameterName.indexOf(":") + 1);
                         parameterName = parameterName.substring(0, parameterName.indexOf(":"));
                     }
-
-                    // Get the search parameter from our filtered set of applicable SPs for this resource type.
-                    SearchParameter searchParameter = applicableSPs.get(parameterName);
-                    if (searchParameter == null) {
-                        String msg =
-                                "Search parameter '" + parameterName + "' for resource type '"
-                                        + resourceType.getSimpleName() + "' was not found.";
-                        throw SearchExceptionUtil.buildNewInvalidSearchException(msg);
+                    
+                    SearchParameter searchParameter = null;
+                    if (isMultiResTypeSearch) {
+                      // Find the SearchParameter that will apply to all the resource types.
+                      for (String resType: resourceTypes) {
+                          Map<String, SearchParameter> resTypeSPs = getApplicableSearchParametersMap(resType);
+  
+                          // Get the search parameter from our filtered set of applicable SPs for this resource type.
+                          searchParameter = resTypeSPs.get(parameterName);
+                          if (searchParameter == null) {
+                              String msg =
+                                      "Search parameter '" + parameterName + "' for resource type '"
```

Comment:
please change the references to parameterName to parameterCode. 
We pass in the code (it may map to name in most cases, however is not consistent with the change from dstu2 to r4.

Reply:
sound good, done